docs/UserGuide.md

```diff
@@ -68,25 +68,32 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
 ## Quick start
 
 1. Ensure you have Java `11` or above installed in your Computer.
+    1. If you have installed Java before, check that you have the right version,  which is Java `11`.
+        - If you are using Windows, open up command prompt and type `java -version` and enter. 
+        - If you are using Mac, open up terminal and type `java -version` and enter.   
+    2. If you do not have Java `11`, you can install it from [here](https://www.oracle.com/java/technologies/downloads/).     
+    <br />
+        
+2. Download the latest Coydir v1.3.0 jar file from [here](https://github.com/AY2223S1-CS2103T-T15-1/tp/releases).
```

Comment:
It'll be `v.1.4.0` or maybe `v1.4.1` later on, but actually I think it's fine to just say "latest version".

Reply:
yes